model-optimizer/extensions/front/kaldi/replace_dropoutmask.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from extensions.middle.MakeKaldiConstReshapable import create_const_with_batch_from_input
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph
+
+
+class ReplaceDropoutMaskPattern(FrontReplacementPattern):
+    enabled = True
+    run_non_recursive = True
+
+    def run_after(self):
+        from extensions.front.restore_ports import RestorePorts
+        return [RestorePorts]
+
+    def run_before(self):
+        from extensions.front.kaldi.replace_lstm_nonlinearity import ReplaceLstmNonLinearityPattern
+        return [ReplaceLstmNonLinearityPattern]
+
+    def find_and_replace_pattern(self, graph: Graph):
+        batch_port = graph.get_op_nodes(op="Parameter")[0].out_port(0)
+        replace_nodes = graph.get_op_nodes(op='dropoutmaskcomponent')
```

Comment:
should we track connection between `dropoutmaskcomponent` and `Parameter` nodes? 
should we replace this approach with a concrete graph pattern?
can we have several `Parameter` nodes with different first dimension?

Reply:
Node dropoutmaskcomponent and Parameter are not connected and it is not important here, because batch in whole graph from Kaldi is the same. We just take it in Parameter because each graph have Parameter and we don't create cycle in this case.
We can't have parameter with different batches, but we can add check here (actually we already check in another transformation).